python_ta/transforms/setendings.py

```diff
@@ -65,13 +66,15 @@
     astroid.DelAttr,
     astroid.Delete,
     # TODO: missing right }
+    # [This one is tricky because there is no way to capture the last brace location]
```

Comment:
This comment and the one below should be removed, now that we have adopted the approach of using the source code to fix them. In fact, all of the TODOs should be handled in this way, so putting in these comments in only two places is kind of misleading.